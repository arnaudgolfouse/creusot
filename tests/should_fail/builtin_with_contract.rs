extern crate creusot_contracts;
use creusot_contracts::*;

#[logic]
#[ensures(true && false)]
#[builtin("dummy_function")]
pub fn builtin_with_contract() {}
