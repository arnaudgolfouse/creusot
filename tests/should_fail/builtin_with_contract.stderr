error: cannot specify both `creusot::builtins` and a contract on the same definition
 --> builtin_with_contract.rs:8:1
  |
8 | pub fn builtin_with_contract() {}
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 1 previous error

